### PR TITLE
Add San Francisco page and team

### PR DESCRIPTION
**Pull request by pletcher**

### CODE REVIEW

File: _layouts/city.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {% include _head.html %}
 {% include _main_nav.html %}
-<header class="site_header site_header_img site_header_{% if page.class %}{{ page.class }}{% else %}{{page.title | downcase}}{% endif %}">
+<header class="site_header site_header_img site_header_{% if page.class %}{{ page.class }}{% else %}{% if page.underscore_title %}{{page.underscore_title}}{% else %}{{page.title | downcase}}{% endif %}{% endif %}">
```
Comment (Ivoz) [inline _layouts/city.html]: Hi, why not just use an extra filter? `| replace: ' ','_'` should work, right?

[liquid reference](https://github.com/Shopify/liquid/wiki/Liquid-for-Designers) if you haven't worked with it much.

**Comment by Ivoz** [comment]
Hi there, sorry for never getting to this :/

If you could rebase and address the comment, I'll get to merging asap.

**Comment by pletcher** [comment]
Hi @Ivoz, no worries! Not sure why we didn't use a filter -- I rebased and fixed that.

Thanks!

File: sass/imports/_main.scss [hunk 2]
```diff
@@ -521,6 +521,9 @@ ul.switch_lang {
     text-align: center;
     &.member {
       @include min-height(240px);
+          height: 220px !important;
+          min-height: 220px;
+          margin-bottom: 40px;
```
Comment (Ivoz) [inline sass/imports/_main.scss]: Could you comment what these are for?

File: _data/cities.yml [hunk 3]
```diff
@@ -88,8 +88,8 @@
   does:
     workshops
 - title: Tel-Aviv
-  meetup_key: opentechschool-tel-aviv
   location: "Tel-Aviv, Isreal"
+  meetup_key: opentechschool-tel-aviv
   meetup: http://www.meetup.com/opentechschool-tel-aviv/
```
Comment (stefanhoth) [inline _data/cities.yml:94]: Could you revert this change?

File: stylesheets/screen.css [hunk 4]
```diff
@@ -189,7 +189,7 @@ ul.switch_lang li.active a { background-color: #085987; color: #FFF; }
 
 /* List of OTS members, extends float list. Includes image and links. */
 .team_list > li { min-height: 8em; height: auto !important; height: 8em; text-align: center; }
-.team_list > li.member { min-height: 240px; height: auto !important; height: 240px; }
+.team_list > li.member { height: 220px !important; min-height: 220px; margin-bottom: 40px; }
```
Comment (stefanhoth) [inline stylesheets/screen.css]: This changes the member listing for all chapters. Are you sure you wanted to do that?

**Comment by stefanhoth** [comment]
Your Twitter avatar changed the color of the OpenTechSchool blue to something else. Could you fix that, please?

File: _data/cities.yml (hunk 3 above)
Comment (pletcher) [inline _data/cities.yml:94]: Of course. Do you mean just moving it back above `location`? It seemed like the others were alphabetical.

File: stylesheets/screen.css (hunk 4 above)
Comment (pletcher) [inline stylesheets/screen.css]: When we originally opened this PR, yes. It looks like it can be safely removed now, though.

**Comment by pletcher** [comment]
Re: Twitter -- sure, but I'm not sure what you mean. It looks to be the same blue as the other Twitter avatars. And I just used the digital color meter -- it's exactly the same blue as https://twitter.com/opentechschool

**EDIT**
Ah, I see what you're saying. That is truly odd. Let me push up a new asset.

File: _data/cities.yml (hunk 3 above)
Comment (stefanhoth) [inline _data/cities.yml:94]: It may be a bit overly control freakish but I want to avoid changes that aren't necessary.

File: _data/members.yml [hunk 5]
```diff
@@ -119,6 +119,19 @@ RheinMain:
...
+    img: chuck.jpg
+  - name: Peter
+    email: peter@opentechschool.org
+    img: peter.jpg
+  - name: Sam
+    email: sam@opentechschool.org
+    img: sam.jpg
+  - name: Will
+    email: will@opentechschool.org
+    img: will.jpg
```
Comment (stefanhoth) [inline _data/members.yml:137]: I saw that your addresses have not been set up yet. Did you send a message to team.tech@opentechschool.org already with your full names and your alternative mail address so we can open the accounts?

File: sass/imports/_main.scss [hunk 6]
```diff
@@ -520,7 +520,11 @@ ul.switch_lang {
     @include min-height(8em);
     text-align: center;
     &.member {
+      // fix spacing for teams that take up more than one line
       @include min-height(240px);
+          height: 220px !important;
+          min-height: 220px;
+          margin-bottom: 40px;
```
Comment (Ivoz) [inline sass/imports/_main.scss]: Hey, it seems you reverted the changes in the compiled css but not the source.

Could you just leave out the changes to this css out of this PR for the moment? (i.e just get the city page & content in)

I can't exactly see what's exactly wrong with the height for the moment but if you think there's still something to be improved here please submit it in another PR and we can get to it that way.

Comment (pletcher) [inline sass/imports/_main.scss]: Absolutely.

Sorry that this PR has been such a mess -- I'll have to admit that many of the changes that are coming in weren't made by me, but were merged locally on our end before I submitted the PR. I should have taken a closer look at things.

File: _data/members.yml (hunk 5 above)
Comment (pletcher) [inline _data/members.yml:137]: I'll get on that -- we weren't sure how to go about setting that up (although I did send an email when we first opened this PR).

**Comment by pletcher** [comment]
Quick question re: the failing Travis builds: Why is Gemfile.lock in the .gitignore? Don't we want that to be checked into version control to make sure that everyone is working with the right versions of everything?

**Comment by Ivoz** [comment]
Looks good enough for now! Thanks being on point touching up the PR, @Pletcher 

I've removed the lock ignore
